cmd/git-init/main.go

```diff
@@ -66,10 +68,30 @@ func main() {
 		logger.Fatalf("Unexpected error creating symlink: %v", err)
 	}
 
+	dir, err := os.Getwd()
+	if err != nil {
+		logger.Errorf("Failed to get current dir", err)
+	}
+
+	if *name != "" {
+		// create dir name
+		path := filepath.Join(dir, *name)
```

Comment:
This doesn't seem to be doing anything... How is `os.Stat(path)` different from `os.Stat(*name)`?